content/docs/pipelines/sdk/python-based-visualizations.md

```diff
@@ -0,0 +1,177 @@
++++
+title = "Python Based Visualizations"
+description = "Predefined and arbitrary visualizations of pipeline outputs"
+weight = 80
++++
+
+This page describes python based visualizations, how to create them, and how to
```

Comment:
As a reader, I wonder whether these Python-based visualisations will eventually replace the visualisations described in the [other doc](https://www.kubeflow.org/docs/pipelines/sdk/output-viewer/) or whether they provide an alternative method, or whether I'm free to use both methods at once, perhaps for different reasons. It'd be good to add a bit of explanation at the top to address these questions.

Also, add info on which version of Kubeflow (e.g. v0.6 and later?) supports the new visualization method.